test/iron-image.html

```diff
@@ -26,6 +46,10 @@
         <iron-image></iron-image>
       </template>
     </test-fixture>
+
+    <div id="fixed-width-container"></div>
```

Comment:
Can we express these as `test-fixture` chunks (like `#TrivialImage` above)? This will help us to avoid leaky state in case we need to add more tests that use these nodes in the future.